docs/10-WebServices.md

```diff
@@ -41,20 +63,38 @@ $I = new ApiTester($scenario);
 $I->wantTo('create a user via API');
 $I->amHttpAuthenticated('service_user', '123456');
 $I->haveHttpHeader('Content-Type', 'application/x-www-form-urlencoded');
-$I->sendPOST('users', ['name' => 'davert', 'email' => 'davert@codeception.com']);
+$I->sendPOST('/users', ['name' => 'davert', 'email' => 'davert@codeception.com']);
 $I->seeResponseCodeIs(200);
 $I->seeResponseIsJson();
 $I->seeResponseContains('{"result":"ok"}');
 ?>
 ```
 
-REST module is designed to be used with services that serve responses in JSON format. For example, method `seeResponseContainsJson` will convert provided array to JSON and check whether response contains it.
+### Testing JSON Responses
+
+The latest line checked that the response contained provided string. However we shoudn't rely on it, as dpending on content formatting we can receive different results with the same data. What we actually need it to check that response can be parsed and it contains some of values we expect. In case of JSON we can use `seeResponseContainsJson` method
```

Comment:
the response contained **the** provided string

as d**e**pending on content formatting

What we actually need **~~it~~is** to check that **the** response can be parsed and it contains some of **the** values we expect. In **the** case of